app/code/community/Oyst/OneClick/etc/system.xml

```diff
@@ -652,6 +654,29 @@
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
                         </position>
+
+                        <!-- class -->
+                        <class translate="label">
```

Comment:
remove this method, duplicate with https://github.com/oystparis/oyst-1click-magento/pull/252/files